temporalcli/commandsgen/commands.yml

```diff
@@ -197,7 +197,8 @@ commands:
         description: Raw payload output, even if the JSON option was used.
       - name: command-timeout
         type: duration
-        description: Timeout for the span of a command.
+        description: |
+          The command execution timeout in seconds. 0 means no timeout.
```

Comment:
This is not "in seconds", you have to provide the unit. You can provide `10ms` for instance. I don't think `0` is an acceptable value unless I'm misreading the code.

Reply:
Amusingly enough, `0` does work, but `5` doesn't. Probably because it doesn't matter what the units are if you specify `0`; it's still no time. But yes, you're right, probably we should show the units and remove "in seconds".

Reply:
Yup, 0 works: https://cs.opensource.google/go/go/+/refs/tags/go1.23.2:src/time/format.go;l=1631-1634

Reply:
fixed